crypto/sig_abstractions_test.go

```diff
@@ -33,17 +33,20 @@ func TestInvalidSinger(t *testing.T) {
 	sigAlgo.Type = MaxAlgorithmType
 
 	dummyMsg := make([]byte, 6)
-	a.Equal(ByteSignature{}, sigAlgo.GetSigner().SignBytes(dummyMsg))
+	_, err = sigAlgo.GetSigner().SignBytes(dummyMsg)
+	a.Error(err)
 
 	dummySig := make([]byte, 6)
 	a.Error(sigAlgo.GetSigner().GetVerifyingKey().GetVerifier().VerifyBytes(dummySig, dummyMsg))
 
-	sigAlgo, err = NewSigner(DilithiumType)
+	sigAlgo, err = NewSigner(FalconType)
 	a.NoError(err)
 	sigAlgo.Type = MaxAlgorithmType
 
 	dummyMsg = make([]byte, 6)
-	a.Equal(ByteSignature{}, sigAlgo.GetSigner().SignBytes(dummyMsg))
+
+	_, err = sigAlgo.GetSigner().SignBytes(dummyMsg)
+	a.Error(err)
```

Comment:
Does it make sense to check the error here?

Reply:
this might be tricky because the signer is an abstraction and the error might be specific to the scheme.